src/kaleidoscope_internal/sketch_exploration/keymap_exploration.h

```diff
@@ -18,8 +18,8 @@
 
 #include "kaleidoscope/key_defs.h"
 
-namespace kaleidoscope {
-namespace sketch_exploration {
+namespace kaleidoscope { // NOLINT(build/namespaces)
+namespace sketch_exploration { // NOLINT(build/namespaces)
```

Comment:
I _think_ cpplint complains here because the path of the file and the namespace doesn't match. However, this file is meant to be used in context of `namespace kaleidoscope`, despite being physically elsewhere. We don't really want to make this non-internal, nor do we want to adjust namespaces (because that'd make code awkward elsewhere), so lets just tell cpplint to ignore it.